dlc-test/src/test/scala/org/bitcoins/dlc/BinaryOutcomeDLCClientIntegrationTest.scala

```diff
@@ -347,6 +349,98 @@ class BinaryOutcomeDLCClientIntegrationTest extends BitcoindRpcTest {
     } yield (acceptDLC, acceptSetup, offerDLC, offerSetup)
   }
 
+  def executeMutualForCase(
+      outcomeHash: Sha256DigestBE,
+      local: Boolean): Future[Assertion] = {
+    val oracleSig =
+      Schnorr.signWithNonce(outcomeHash.bytes, oraclePrivKey, preCommittedK)
+
+    val setupsAndDLCs = for {
+      (acceptDLC, acceptSetup, offerDLC, offerSetup) <- constructAndSetupDLC()
+    } yield {
+      if (local) {
+        (offerDLC, offerSetup, acceptDLC, acceptSetup)
+      } else {
+        (acceptDLC, acceptSetup, offerDLC, offerSetup)
+      }
+    }
+
+    val outcomeFs = setupsAndDLCs.map {
+      case (initDLC, initSetup, otherDLC, otherSetup) =>
+        val closeSigsP = Promise[(SchnorrDigitalSignature, PartialSignature)]()
+        val initSendSigs = {
+          (sig: SchnorrDigitalSignature, fundingSig: PartialSignature) =>
+            closeSigsP.success(sig, fundingSig)
+            FutureUtil.unit
+        }
+
+        val mutualCloseTxP = Promise[Transaction]()
+
+        val watchForMutualCloseTx = new Runnable {
+          override def run(): Unit = {
+            if (!mutualCloseTxP.isCompleted) {
+              clientF.foreach { client =>
+                val fundingTxResultF = client.getRawTransaction(
+                  initDLC
+                    .createUnsignedMutualClosePSBT(oracleSig,
+                                                   initSetup.fundingTx)
+                    .transaction
+                    .txIdBE)
+
+                fundingTxResultF.onComplete {
+                  case Success(fundingTxResult) =>
+                    if (fundingTxResult.confirmations.isEmpty) {
+                      ()
```

Comment:
Error message here?

Reply:
No need to fail, if no confirmations then nothing is done and another call is made after `1.second`